cmd/secrets/list.go

```diff
@@ -0,0 +1,31 @@
+package secrets
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	. "github.com/tensorleap/leap-cli/pkg/api"
+	"github.com/tensorleap/leap-cli/pkg/auth"
+)
+
+func init() {
+	RootCommand.AddCommand(&cobra.Command{
+		Use:   "list",
+		Short: "List available secrets",
+		Long:  `List available secrets`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := auth.CheckLoggedIn(); err != nil {
+				return err
+			}
+			data, _, err := ApiClient.GetSecretManagerList(cmd.Context()).Execute()
+			if err != nil {
+				return err
+			}
+			for _, secret := range data.Results {
+				fmt.Println(secret.GetName(), "-", secret.GetCid())
```

Comment:
```suggestion
				log.Println(secret.GetName(), "-", secret.GetCid())
```
Import and use our logger so this will also be cataloged to the local log file